build.sbt

```diff
@@ -35,7 +35,8 @@ lazy val compilerOpts = Seq(
   "-Ywarn-unused",
   "-unchecked",
   "-deprecation",
-  "-feature"
+  "-feature",
+  "-Xsource:2.12"
```

Comment:
FYI

https://github.com/scala/scala/blob/2f1b5259188698501dbc8430f63972bf7bc68154/src/compiler/scala/tools/nsc/settings/ScalaSettings.scala#L138

Reply:
Actually ended up posting a question Scala Users: https://users.scala-lang.org/t/understanding-the-xsource-option/4749